src/components/styled-components/Typography.js

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components';
+
+export default function Typography({ children, variant, component, ...rest }) {
+	switch (variant) {
+		case 'h1':
+			return (
+				<StyledH1 {...rest} as={component}>
+					{children}
+				</StyledH1>
+			);
+		case 'h2':
+			return (
+				<StyledH2 {...rest} as={component}>
+					{children}
+				</StyledH2>
+			);
+		case 'h3':
+			return (
+				<StyledH3 {...rest} as={component}>
+					{children}
+				</StyledH3>
+			);
+		case 'h4':
+			return (
+				<StyledH4 {...rest} as={component}>
+					{children}
+				</StyledH4>
+			);
+		case 'h5':
+			return (
+				<StyledH5 {...rest} as={component}>
+					{children}
+				</StyledH5>
+			);
+		case 'h6':
+			return (
+				<StyledH6 {...rest} as={component}>
+					{children}
+				</StyledH6>
+			);
+		default:
+			return (
+				<StyledP {...rest} as={component}>
+					{children}
+				</StyledP>
+			);
+	}
+}
+
```

Comment:
This is the first file in your styled-components section, that makes sense to me. Please take a look again at the other files.